client/app/helpers/format-date.coffee

```diff
@@ -1,10 +1,11 @@
 `import Ember from 'ember'`
 
-formatDate = (date, options) ->
+formatDate = (date, format) ->
```

Comment:
hi @4thethrillofit this change worked for you?  since this was my first attempt but later with @egh we found that all helpers expect 2 arguments, the last parameter is an options object which have a "hash" key in which wrapped the values sent  :smiley: